automatic/kitty/tools/helpers.ps1

```diff
@@ -0,0 +1,20 @@
+﻿function StopProcesses() {
+  $processNames = @("genpass", "gensfc", "kageant", "cygtermd", "kitty", "kittygen", "klink", "kscp", "ksftp")
+  $processes = Get-Process  $processNames -ea SilentlyContinue
+
+  if ($processes) {
```

Comment:
Reverse this logic please, so we don't have rest of the function idented.

Reply:
Function simplified